### ISSUE TITLE
[Feature] add support for unittest.TestCase fixtures

### ISSUE THREAD

**Issue by Ivaylo-Bachvarov** [microsoft/playwright-pytest#55]
It would be really helpful to have an example with `unittest.TestCase`

I'm struggling to receive the page object in my test class.

**Comment by mxschmitt** [comment]
Hi, its not yet possible. Only function based tests are currently implemented.

(It would require for us to do something like and using the [class scope](https://docs.pytest.org/en/6.2.x/unittest.html#mixing-pytest-fixtures-into-unittest-testcase-subclasses-using-marks).)

**Comment by Ivaylo-Bachvarov** [comment]
Thanks for the quick replay! 

I will fork and give it a shot. If it works I will contribute back.

**Comment by Ivaylo-Bachvarov** [comment]
Here is what I did:

```

class BasePlaywrightTestCase(TestCase):
    @pytest.fixture(autouse=True)
    def init_playwright(self, page, browser):
        self.page = page
        self.browser = browser


class ExampleTest(BasePlaywrightTestCase):
    def test_one(self):
        print(self.page)
```

This is inspired by the py.test documentation. 

The error that I hit was:
```
    @pytest.fixture(scope="session")
    def browser_type(playwright: Playwright, browser_name: str) -> BrowserType:
>       return getattr(playwright, browser_name)
E       TypeError: getattr(): attribute name must be string
```

Here is how I fixed it. I know this fix breaks support for multiple browser runs. But that is all I managed to do for now. It works for me since we test only on chrome for now. I'm just linking it here in case it is helpful for someone.

https://github.com/Ivaylo-Bachvarov/playwright-pytest/pull/1/files

**Comment by mxschmitt** [comment]
Hey @Ivaylo-Bachvarov I created a PR, feel free to go over it which should fix your issue: #56

There is unfortunately an upstream limitation on pytest side, that parameterize the tests using multiple browser arguments does not work. We now print a warning if its > 1. Use by default chromium if its 0 and when its 1 I use the specified one.

**Comment by mxschmitt** [comment]
I released version 0.1.2 which should fix it.

**Comment by Ivaylo-Bachvarov** [comment]
You are great! Thanks!
